app/stac_api/collection_temporal_extent.py

```diff
@@ -173,6 +199,9 @@ def update_end_temporal_extent_on_item_update(
             the id of the item being treated (pk)
         qs: QuerySet | None
```

Comment:
maybe renaming `qs` with `qs_items` or `qs_other_items` would make the code clearer

Reply:
good point, I'll change the naming